python_modules/dagster-graphql/dagster_graphql/implementation/execution/backfill.py

```diff
@@ -114,11 +114,14 @@ def create_and_launch_partition_backfill(
     partitions_by_assets = backfill_params.get("partitionsByAssets")
 
     if (
-        asset_selection or backfill_params.get("selector") or backfill_params.get("partitionNames")
+        asset_selection
+        or backfill_params.get("selector")
+        or backfill_params.get("partitionNames")
+        or backfill_params.get("allPartitions")
     ) and partitions_by_assets:
         raise DagsterInvariantViolationError(
-            "partitions_by_assets cannot be used together with asset_selection, selector, or"
-            " partitionNames"
+            "partitionsByAssets cannot be used together with assetSelection, selector, or"
+            " partitionNames or if allPartitions is True"
```

Comment:
I think it makes sense to validate `allPartitions`, too, if checking `partitionNames`; neither can be used with `partitionsByAssets`.

I would be happy to split this and the above change into a separate (stacked?) PR, since they're not directly related to creating the Python API; just thought I'd do some housekeeping as I was learning about it for the Python API implementation.